pom.xml

```diff
@@ -1,12 +1,13 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>com.axonivy.market</groupId><!-- your group id: e.g. com.axonivy.connector.<myconnector> or com.axonivy.utils.<myutil> -->
-  <name>my-product</name><!-- fill your product name -->
-  <artifactId>my-product-modules</artifactId><!-- fill your product name + a "-modules" postfix -->
-  <version>10.0.0-SNAPSHOT</version><!-- identicate your minimal compliant ivy version with the first 2 digits -->
+  <groupId>com.axonivy.connector.ups</groupId><!-- your group id: e.g. com.axonivy.connector.<myconnector> or com.axonivy.utils.<myutil> -->
+  <name>ups-connector</name><!-- fill your product name -->
+  <artifactId>ups-connector-modules</artifactId><!-- fill your product name + a "-modules" postfix -->
+  <version>10.0.14-SNAPSHOT</version><!-- identicate your minimal compliant ivy version with the first 2 digits -->
```

Comment:
please remove the hint in this pom? that hint supports we input the value after finishing we should remove it. Thanks!